downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilder.java

```diff
@@ -55,16 +57,18 @@ public static void processCellularComponents(GKInstance reactionInst) throws Exc
      * @param taxonIdentifier -- String, CrossReference ID of protein's species.
      * @throws Exception -- MySQLAdaptor exception.
      */
-    private static void generateGOCellularCompartmentLine(GKInstance proteinInst, GKInstance reactionInst, String taxonIdentifier) throws Exception {
+    private static String generateGOCellularCompartmentLine(GKInstance proteinInst, GKInstance reactionInst, String taxonIdentifier) throws Exception {
         String reactomeIdentifier = REACTOME_IDENTIFIER_PREFIX  + GOAGeneratorUtilities.getStableIdentifierIdentifier(reactionInst);
         GKInstance referenceEntityInst = (GKInstance) proteinInst.getAttributeValue(ReactomeJavaConstants.referenceEntity);
         String goCellularCompartmentAccession = getCellularCompartmentGOAccession(proteinInst);
+        String goaLine = null;
```

Comment:
It is better to assign an empty String rather than null unless there's a strong reason not to so you can avoid having to do null checks and risking a NullPointerException being thrown.